### DISCUSSION TITLE
Complete Tweet, Language and Location

### DISCUSSION THREAD

**Discussion by JijoC-13** [cjbarrie/academictwitteR#106] [Q&A]
Dear Team,

Firstly, on inspection I found that the retrieved tweets by the code "get_all_tweets" are not complete. It only covers up-to what the "Keyword" of the query is. 
Is there a way around this or am I not doing something right? as I am interested in the entire tweet to do a sentimental analysis.

Secondly, I specified the language to be "en" and still see a couple of tweets containing other languages. Why does that happen?

Thirdly, specified country is also not captured. How do I specify only those tweets from a specific country and make sure it is from there?

Looking forward to your response.
Thanks in advance.

**Comment by cjbarrie** [comment]
Ji @JijoC-13 it's difficult for us to help unless you share your code. Please provide the code you used when you encountered this error, the version of the package you're using etc. This is really for the Issues page as well not the Discussions page. Feel free to open an Issue following the template provided

**Reply by JijoC-13** [reply]
Thanks for the feedback.

Sorry for the wrong posting. New to this portal.
